server/tests/src/test/java/org/infinispan/server/functional/RollingUpgradeIT.java

```diff
@@ -171,7 +170,7 @@ private void createSourceClusterCache(RestClient client) {
 
    private static class ClusterConfiguration extends InfinispanServerTestConfiguration {
       public ClusterConfiguration(String configurationFile, int numServers, int portOffset) {
-         super(configurationFile, numServers, ServerRunMode.EMBEDDED, new Properties(), null, null,
```

Comment:
Isn't this kind of excessive?

Same with https://github.com/infinispan/infinispan/blob/b23ab3509ab93be12d820fdadc5be350fb772cca/server/testdriver/core/src/main/java/org/infinispan/server/test/core/AbstractServerConfigBuilder.java#L33

Reply:
What do you propose?